templates/ts-apollo-postgres-backend/src/index.ts

```diff
@@ -26,7 +27,10 @@ const db = connectDB()
 const dbConfig = loadDBConfig()
 
 const { typeDefs, resolvers, contextCreator } = buildGraphbackAPI(modelDefs, {
-  dataProviderCreator: createKnexDbProvider(db)
+  dataProviderCreator: createKnexDbProvider(db),
+  plugins: [
+    new ClientCRUDPlugin({ outputFile: 'doc.ts' })
```

Comment:
Makes no sense. I verified this working, but should be removed from template

Reply:
Sorry, this was me doing manual testing